### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by vdubr**
- update i18n packages 
- change exported i18n API [withNamespaces -> withTranslation]
- update packages

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,31 +25,30 @@
 		"url": "https://github.com/gisat/ptr-locales"
 	},
 	"dependencies": {
-		"i18next": "^19.8.7",
-		"lodash": "^4.17.20",
+		"i18next": "^20.3.2",
+		"lodash": "^4.17.21",
 		"react": "16.13.1",
-		"react-i18next": "9.0.10"
+		"react-i18next": "^11.11.0"
```
Comment (vlach1989) [inline package.json:31]: update react to ^16.13.1

File: package.json [hunk 2]
```diff
@@ -25,31 +25,30 @@
...
-		"prettier": "^2.2.1",
-		"pretty-quick": "^3.1.0",
+		"prettier": "^2.3.1",
+		"pretty-quick": "^3.1.1",
 		"rimraf": "^3.0.2",
-		"rollup": "^2.39.0",
+		"rollup": "^2.52.1",
 		"rollup-plugin-babel": "^4.4.0",
-		"rollup-plugin-filesize": "^9.1.0",
+		"rollup-plugin-filesize": "^9.1.1",
```
Comment (vlach1989) [inline package.json:51]: migrate to @rollup/plugin-babel

**Review by vlach1989 — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (vdubr) [reply package.json]: Yep.
Fixed: 5f301e990879ef53b30da10804b4187d05edc415

File: package.json (hunk 2 above)
Comment (vdubr) [reply package.json]: fixed: 7644deab652196c9bd02aea9c101c3e5206dced2

**Review by vlach1989 — APPROVED** [review]

**Pull request merged**